bin/deploy-staging

```diff
@@ -7,22 +7,48 @@ set +x
 export AWS_DEFAULT_REGION=us-west-2
 REGION=us-west-2
 
-aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws/t1q6b4h2
-docker build -f prod.Dockerfile -t universal-application-tool --cache-from docker.io/civiform/civiform-browser-test:latest --build-arg BUILDKIT_INLINE_CACHE=1 .
-docker tag universal-application-tool:latest public.ecr.aws/t1q6b4h2/universal-application-tool:latest
+aws ecr-public get-login-password --region us-east-1 |
+  docker login --username AWS --password-stdin public.ecr.aws/t1q6b4h2
+
+docker build \
+  -f prod.Dockerfile \
+  -t universal-application-tool \
+  --cache-from docker.io/civiform/civiform:latest \
+  --build-arg BUILDKIT_INLINE_CACHE=1 \
```

Comment:
💁  changing the `--cache-from` flag value from `docker.io/civiform/civiform-browser-test:latest` to `docker.io/civiform/civiform:latest` should be the only change to behavior in this PR